cargo-espflash/README.md

```diff
@@ -27,26 +27,24 @@ Supports the **ESP32**, **ESP32-C2/C3/C6**, **ESP32-H2**, **ESP32-P4**, and **ES
 
 If you are installing `cargo-espflash` from source (ie. using `cargo install`) then you must have `rustc>=1.74.0` installed on your system.
 
-If you are running **macOS** or **Linux** then [libuv] must also be installed; this is available via most popular package managers. If you are running **Windows** you can ignore this step.
+If you are running **Linux** then [libudev] must also be installed; this is available via most popular package managers. If you are running **Windows** you can ignore this step.
 
-```bash
-# macOS
-brew install libuv
+```console
```

Comment:
Is there any reason to change this to `console`?

Reply:
OK, back to using bash here.